packages/experiment-browser/src/factory.ts

```diff
@@ -71,7 +71,14 @@ const newExperimentClient = (
 ): ExperimentClient => {
   return new ExperimentClient(apiKey, {
     ...config,
-    userProvider: new DefaultUserProvider(config?.userProvider, apiKey),
+    userProvider: config?.['consentAwareStorage']
+      ? new DefaultUserProvider(
+          config?.userProvider,
+          apiKey,
+          config?.['consentAwareStorage']?.['localStorage'],
+          config?.['consentAwareStorage']?.['sessionStorage'],
+        )
+      : new DefaultUserProvider(config?.userProvider, apiKey),
```

Comment:
nit: can be simplified to

```suggestion
    userProvider: new DefaultUserProvider(
      config?.userProvider,
      apiKey,
      config?.['consentAwareStorage']?.localStorage,
      config?.['consentAwareStorage']?.sessionStorage,
    ),
```